operations/helm/charts/mimir-distributed/values.yaml

```
@@ -551,6 +551,11 @@ alertmanager:
   #   E.g. if 'replicas' is set to 4 and there are 3 zones, then 4/3=1.33 and after rounding up it means 2 pods per zone are started.
   replicas: 1
 
+  # -- Allows to override the container image of the alertmanager component.
```

ok, i may be getting annoying, but can you mention that this takes precedence over the root-level `image` and `enterprise.image` sections?


```suggestion
  # -- Allows to override the container image of the alertmanager component from the root-level `image` and `enterprise.image` sections.
```

something along those lines. You should be able to do a find-and-replace of the string.